test/cpp/modules/common/GmmTests.cpp

```diff
@@ -22,6 +22,8 @@ TEST_P(GmmModuleTest, Load)
     ASSERT_NE(module, nullptr);
 }
 
+const double epsilon = 1e-08;
```

Comment:
Did you mean to remove this const after introducing a test parameter for tolerance?